astronomer/providers/google/cloud/operators/dataproc.py

```diff
@@ -295,61 +100,16 @@ def __init__(
         polling_interval: float = 5.0,
         **kwargs: Any,
     ):
-        super().__init__(**kwargs)
+        warnings.warn(
+            (
+                "This module is deprecated and will be removed in 2.0.0."
+                "Please use `airflow.providers.google.cloud.operators.dataproc.DataprocUpdateClusterOperator`"
+                "and set `deferrable` param to `True` instead."
+            ),
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        super().__init__(deferrable=True, **kwargs)
```

Comment:
look like we polling_interval_seconds in kwargs https://github.com/apache/airflow/blob/main/airflow/providers/google/cloud/operators/dataproc.py#L2452

Reply:
yep, miss this one. just updated it. thanks!